dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/AnonymisingDataService.java

```diff
@@ -0,0 +1,281 @@
+package uk.gov.justice.laa.crime.dces.integration.service;
+
+import lombok.extern.slf4j.Slf4j;
+import net.datafaker.Faker;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import uk.gov.justice.laa.crime.dces.integration.model.generated.contributions.CONTRIBUTIONS;
+
+import java.util.Optional;
+import java.util.Random;
+
+import static uk.gov.justice.laa.crime.dces.integration.utils.DateConvertor.convertToXMLGregorianCalendar;
+
+@Slf4j
+@Service
+public class AnonymisingDataService {
+
+    private static final long DEFAULT_SEED_VALUE = 38;
+    private Faker faker;
+
+    public CONTRIBUTIONS anonymise(CONTRIBUTIONS contribution) {
+
+        long seedId = DEFAULT_SEED_VALUE * contribution.getMaatId();
+        log.info("Anonymising data for contribution with maatId {} and seedId: {}", contribution.getMaatId(), seedId);
+        faker = new Faker(new Random(seedId));
+
+
+        if (hasValue(contribution.getMaatId())) {
+            contribution.setMaatId(faker.number().numberBetween(100000, 999999));
+        }
+        Optional.ofNullable(contribution.getApplicant())
+                .ifPresent(applicant -> contribution.setApplicant(anonymiseApplicant(applicant)));
+        Optional.ofNullable(contribution.getApplication())
+                .ifPresent(application -> contribution.setApplication(anonymiseApplication(application)));
+        Optional.ofNullable(contribution.getCapitalSummary())
+                .ifPresent(capitalSummary -> contribution.setCapitalSummary(anonymiseCapitalSummary(capitalSummary)));
+        Optional.ofNullable(contribution.getPassported())
```

Comment:
I know the confluence page says to anonymise this but it shouldn't be. Can you remove it from the code ? I will update the confluence page. this comment is for the ASSESSMENT DETAILS->PASSPORTED